### DISCUSSION TITLE
Unsupported libraries

### DISCUSSION THREAD

**Discussion by flowmeter** [NicoHood/PinChangeInterrupt#51] [Q&A] [answered]
Hello world.
I'm wondering if HX711 library and PinChangeInterrupt should never been used together ?
My setup stops at scale.tare(); 
I'm wondering if it is because i'm using https://github.com/bogde/HX711 library and PinChangeInterrupt together ?
SCA and SCL are the same as A4 and A5, so using PinChangeInterrupt may cause some trouble to HX711
scale.set_scale(calibration_factor); works but i think it sends nothing on SCA & SCL lines.
Any insight ?
Best

**Comment by NicoHood** [comment, marked as answer]
As long as your other library does not use the PCINT itself (with the PCINt functionality itself) it is fine. If you do not set a PCINt on SCA/SCL, it would work. Just use some other pins.

**Reply by flowmeter** [reply]
Thank you Nico for your answer.
Sorry, you cand discard everything I made a mistake, this is RTC that is on SDA SCL not HX711.
My bad.
